### ISSUE TITLE
Accessibility test fails on edit descriptive metadata form

### ISSUE THREAD

**Issue by luis100** [keeps/roda#950]
On descriptive metadata edit form, the accessibility test fails.

![roda - catalogue 5](https://cloud.githubusercontent.com/assets/631728/25494022/34897be2-2b70-11e7-94c9-354ab789e625.jpg)

In portuguese:
http://www.acessibilidade.gov.pt/accessmonitor/?cD00MjIyMDI,&wcag20

Controlos de formulário sem etiquetas [<label>] associadas e sem atributo title

Os elementos <label> associados aos elementos <input> asseguram que a informação relacionada com os campos é lida pelos leitores de ecrã sempre que os campos de edição recebem o foco. O atributo title pode ser usado sempre que seja difícil, em termos de design, acomodar a label ou quando a mesma possa causar confusão.

![accessmonitor](https://cloud.githubusercontent.com/assets/631728/25494116/799296ba-2b70-11e7-9103-88d3fa4e6053.jpg)

**Comment by nunovieira220** [comment]
Solved on b1cb7a27b0daee9c47b4e91c102e7c7857b034fc
